### DISCUSSION TITLE
Unable to establish connection to SQL Server instance

### DISCUSSION THREAD

**Discussion by nareshdba333** [burningalchemist/sql_exporter#647] [Q&A] [answered]
I am facing issue while trying to connect MS SQL Server

![image](https://github.com/user-attachments/assets/f2dae8d4-bf0a-40dc-ac5c-12e8cb139511)

**Comment by burningalchemist** [comment, marked as answer]
Hi @nareshdba333, your syntax is wrong as per the error.
Remove ` = ` to have it as `--config.file 'path'`. Or use `=` without whitespaces. One of these should work.
